stats/agent_handler.go

```diff
@@ -0,0 +1,59 @@
+package stats
+
+import (
+	"context"
+	"git.fd.io/govpp.git/api"
+	"github.com/ligato/cn-infra/logging/logrus"
+	govppcalls "go.ligato.io/vpp-agent/v2/plugins/govppmux/vppcalls"
+	telemetrycalls "go.ligato.io/vpp-agent/v2/plugins/telemetry/vppcalls"
+	ifplugincalls "go.ligato.io/vpp-agent/v2/plugins/vpp/ifplugin/vppcalls"
+)
+
+// AgentHandler uses Ligato VPP-Agent interface and telemetry handlers
+// to obtain data from VPP
+type AgentHandler struct {
```

Comment:
Seems more like `VPPHandler`, since none of the methods have anything to do with an agent.